import-data.sh

```diff
@@ -46,15 +46,17 @@ if [ ! -f "${DATAVERSION_PATH}" ]; then
   DATAVERSION_FLAG=""
 fi
 
-# Import synapse data to database
-# Not using --mode upsert for now because we don't have unique indexes properly set for the collections
+# Drop and recreate the entire database to ensure clean state
+echo "Dropping and recreating model-ad database..."
+mongosh --host $DB_HOST -u $DB_USER -p $DB_PASS --authenticationDatabase admin --eval "db.getSiblingDB('model-ad').dropDatabase()"
```

Comment:
Do you have a sense of how much application downtime this will add to our data deployments?

Reply:
I've updated the approach to look for orphaned collections since I wasn't happy with this approach.  Downtime would be hard to measure at this time but I added logging so execution time is displayed when the script completes.